src/box/iproto.h

```diff
@@ -74,6 +76,25 @@ struct iproto_stats {
 	size_t requests_in_stream_queue;
 };
 
+/**
+ * IPROTO request handler destructor called when the corresponding handler is
+ * removed.
+ */
+typedef void
+(*iproto_rq_handler_dtor_f)(void *ctx);
+
+/**
+ * Request handler meta information.
+ */
+struct iproto_rq_handler {
```

Comment:
AFAICS the struct is used only in the `iproto.cc` so it should be defined there, no in the header.

Reply:
Indeed, initially, I used it, but then it became unnecessary, fixed: moved to `iproto.cc`.